ft232/d2xx.py

```diff
@@ -283,7 +283,7 @@ def setWriteTimeout(self, timeout):
 
     def getWriteTimeout(self):
         """Get the current timeout setting."""
-        return self._writeTimeout
+        return self._writetimeout
```

Comment:
This looks like a bug that exists in d2xx and it looks like there's some more inconsistent usage of writetimeout. 

I'd appreciate it if you can fix all of it and send this as a separate PR.